packages/blocks/src/components/block-hub.ts

```diff
@@ -855,7 +861,11 @@ export class BlockHub extends WithDisposable(ShadowlessElement) {
       >
         ${blockHubMenu}
         <div
-          class="has-tool-tip new-icon ${this._expanded ? 'icon-expanded' : ''}"
+          class="has-tool-tip new-icon ${this._inEdgelessMode
+            ? this._expanded
+              ? 'icon-expanded'
+              : 'new-icon-in-edgeless'
+            : ''}"
```

Comment:
To simplify complex CSS classes, you should consider using the `classMap` function from Lit.